packages/jest-expo/babel.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    // Only use this for the `tests/` folder.
+    env: {
+      test: {
```

Comment:
@ide this seems like a clear solution https://github.com/facebook/jest/issues/3845
But unsure how we want to test usage outside the repo.

Reply:
This looks good. For outside the workspace, one smoke test is in the tabs template project that has a couple tests (including component tests).

Reply:
This appears to work as expected